### ISSUE TITLE
Not compatible with NextJs12

### ISSUE THREAD

**Issue by Tagir-A** [panelbearhq/panelbear-nextjs#1]
Hey there 👋 !

I've been trying to use the official integration with my project in NextJs12, but it gives an error.
I'm pretty sure it's cause the panelbear plugin expects previous versions of NextJs.

Here's the error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: bk-korona-rate@undefined
npm ERR! Found: next@12.0.2
npm ERR! node_modules/next
npm ERR!   next@"12.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer next@"^10 || ^11" from @panelbear/panelbear-nextjs@1.0.1
npm ERR! node_modules/@panelbear/panelbear-nextjs
npm ERR!   @panelbear/panelbear-nextjs@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/gitpod/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2021-11-04T19_06_34_859Z-debug.log
```

P.S.

Thank you for your product 👍

**Comment by tomelliot** [comment]
I also ran into this. Adding the ` --legacy-peer-deps` flag when installing the package is a workaround.

**Comment by anthonynsimon** [comment]
Hi @tomelliot - thanks for reporting this!

Just released a fix in `1.0.3`. Feel free to check it out!

**Comment by tomelliot** [comment]
Thanks for the quick fix @anthonynsimon !
